ner_v2/detectors/textual/elastic_search.py

```diff
@@ -142,6 +143,12 @@ def get_multi_entity_results(self, entities, texts, fuzziness_threshold=1,
         try:
             response = self._run_es_search(self._default_connection, **kwargs)
             results = _parse_multi_entity_es_results(response.get("responses"))
+        except es_exceptions.NotFoundError as e:
+            raise DataStoreRequestException(f'NotFoundError in datastore query on index: {index_name}', engine='elasticsearch',
+                                            request=json.dumps(data), response=json.dumps(response))
+        except es_exceptions.ConnectionError as e:
+            raise DataStoreRequestException(f'Error while connecting to ES, in datastore query on index: {index_name}',
+                                            engine='elasticsearch', request=json.dumps(data), response=json.dumps(response))
```

Comment:
E501 line too long (124 > 119 characters)

Reply:
resolved in latest commit